html/components/autocomplete.stories.js

```diff
@@ -0,0 +1,240 @@
+        >
+          <svg
+            class="
+              sprk-c-Icon
+              sprk-c-Icon--filled-current-color
+              sprk-c-Icon--stroke-current-color
+            "
+            viewBox="0 0 64 64"
+            aria-hidden="true"
+          >
+            <use xlink:href="#search" />
+          </svg>
+
+          <input
+            class="
+                  sprk-b-TextInput
+                  sprk-b-TextInput--has-svg-icon
+                  sprk-u-Width-100
+                "
+            id="autocomplete-input"
+            type="text"
+            aria-autocomplete="list"
+            aria-describedby="input--error-container"
+            autocomplete="off"
+            autocapitalize="off"
+            spellcheck="false"
+          >
+        </div>
+
+        <ul
+          class="sprk-c-Autocomplete__results sprk-u-Width-100"
+          role="listbox"
+          aria-labelledby="autocomplete-label"
+        >
+          <li class="sprk-c-Autocomplete__result" role="option">Apple</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Apricot</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Avocado</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Banana</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Blackberry</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Blueberry</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Clementine</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Fig</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Grape</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Kumquat</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Lemon</li>
+          <li class="sprk-c-Autocomplete__result" role="option">Papaya</li>
+        </ul>
+
+        <div
+          class="sprk-b-ErrorContainer"
+          id="input--error-container"
+        ></div>
+      </div>
+    </div>
+  `;
+};
+
+defaultStory.story = {
+  name: 'Default',
+  parameters: {
+    jest: ['autocomplete'],
+  },
+};
+
+export const hugeStory = () => {
+  useEffect(() => {
+    autocomplete();
+  }, []);
+
+  return `
+  <div data-sprk-autocomplete="container">
+    <div aria-live="polite" class="sprk-u-ScreenReaderText"></div>
+
+    <div class="sprk-b-InputContainer sprk-b-InputContainer--huge">
+      <div
+        class="sprk-b-TextInputIconContainer"
+        role="combobox"
+        aria-haspopup="listbox"
+        aria-expanded="true"
+        aria-labelledby="input-label-huge"
+      >
+        <svg
+          class="
+            sprk-c-Icon
+            sprk-c-Icon--filled-current-color
+            sprk-c-Icon--stroke-current-color
+          "
+          viewBox="0 0 64 64" aria-hidden="true"
+        >
+          <use xlink:href="#search" />
+        </svg>
+        <input
+          class="sprk-b-TextInput sprk-u-Width-100"
```

Comment:
Maybe we need a sprk-b-TextInput--full class for the 100% width.

Reply:
We currently have `sprk-u-Width-100` in all of the HTML input stories 😟